CONTRIBUTING.md

```diff
@@ -1,143 +1,177 @@
+ln ./pre-commit.sh .git/hooks/pre-commit
+# Lets see how long execution takes
+time .git/hooks/pre-commit
 ```
-
-### Quellcode Formatierung
-
-Wir verwenden php-cs-fixer für PHP und ESLint und Prettier für JS um einen gemeinsamen Code Style zu etablieren. Bitte stelle sicher, dass dein Code automatisch richtig formatiert wird, bevor du diesen ins Git repo committest.
-
-Wir empfehlen deine [IDE so zu konfigurieren](https://github.com/ecamp/ecamp3/wiki/installation-development-windows#code-auto-formatting), dass dein Code beim Speichern automatisch richtig formatiert wird.
-
-Alternativ kannst du
-
-- php-cs-fixer und ESLint / Prettier vor jedem commit manuell laufen lassen:
-  ```bash
-  docker compose run api composer cs-fix
-  docker compose run frontend npm run lint
-  docker compose run print npm run lint
-  ```
-- einen git pre-commit hook einrichten, der php-cs-fixer und ESLint vor jedem commit triggert
-
-### Vor dem Einreichen eines Pull Requests
-
-- [x] Wurden alle geänderten oder neuen PHP-Dateien von cs-fixer verarbeitet?
-- [x] Wurden alle geänderten oder neuen JS/Vue-Dateien von ESLint / Prettier bereinigt?
-- [x] Sind alle Variabeln, Klassen, Funktionen, Kommentare, etc. auf englisch benannt?
-- [x] Hast du für neue Funktionalität Tests geschrieben und für geänderte Funktionalität die Tests angepasst?
-- [x] Wurden alle Passwörter, Zugangsdaten und lokale Konfiguration aus den Code-Änderungen entfernt?
-- [x] Enthalten alle geänderten Dateien auch wirklich eine sinnvolle Änderung (im Gegensatz zu z.B. nur Whitespace-Änderungen)?
-- [x] Ist der Fork auf dem aktuellen Stand des zentralen Repositories und können die Änderungen automatisch gemerged werden?
-- [x] Ist der GitHub Actions CI-Build erfolgreich und ohne Test Failures durchgelaufen?
+</details>
+
+#### Checklist :pencil:
+
+We truly value and appreciate every contribution to our project! :heart:
+To make the collaboration smoother and more enjoyable for everyone, 
+we've put together this checklist :scroll:.
+Following it will not only enhance the quality and consistency of your contributions:sparkles:  but also fast-track the review process. :rocket:
+
+
+- [x] **Sync with Central Repository:** :arrows_counterclockwise: Ensure your fork is up to date with the central repository, facilitating a smooth merge.
+- [x] **Lint:** :wrench: Ensure that linters have run over all new or modified files
+- [x] **Spelling:** :abc: Ensure that the changes are spell checked
+- [x] **Significant Changes:** :mag_right: Confirm that every modified file contributes meaningful content, steering clear of inconsequential changes like mere whitespace adjustments.
+- [x] **Testing:** :test_tube: Write tests for any new features, and update existing ones if you've made changes to functionalities.
+- [x] **Language & Spelling:** :book: Use English for all variable names, class names, functions, comments, etc., and ensure that all added content has been spellchecked.
```

Comment:
Spelling is already mentioned above, remove either one

Reply:
Those two mean slightly different things. The first one is documentation (which I don't think needs to be English) and the other one is code, like variable names (which I do think needs to be (mostly) English).
However, I must admit that this adds no value and is just overly complicated so I will "merge" them.